external/imgui_entt_entity_editor/imgui_entt_entity_editor.hpp

```diff
@@ -1,9 +1,9 @@
 // for the license, see the end of the file
 #pragma once
 
+#include <functional>
```

Comment:
Was this an automatic change or an accident? The change is not really needed

Reply:
cause clang tidy sorts the includes